docker/notebook/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# This Dockerfile containers Jupyter Notebook server with many
```

Comment:
I feel that `jupyter` might be a better name than `notebook` because there could be other kinds of notebooks like Colab notebooks.  What do you think?

Reply:
Thanks for the reminding, I think `jupyter` is okay.
There are also many kinds of Jupyter Notebooks list on https://hub.docker.com/search?q=jupyter&type=image

we can build `sqlflow/sqlflow:jupyterhub` on the multi-user scene in the future.